inspec/a2-api-integration/controls/notifications-alert.rb

```diff
@@ -34,6 +34,24 @@
       end
     end
 
+    #converge-failure-with-ignored-failure-report_run_converge
+    let(:failed_ccr_message_with_ignored_failure) do
+      return @failed_ccr_json ||= begin
```

Comment:
Memoizing with an instance variable inside a `let` block is redundant, and it is dangerous in this case since the instance variable will be set by the first `let` to be referenced.